apps/mdn/k8s/prod/kuma.yaml.j2

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ KUMA_NAME }}
+spec:
+  type: LoadBalancer
+  selector:
+    app: {{ KUMA_NAME }}
+  ports:
+    - port: 80
+      targetPort: 8000
+      protocol: TCP
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ KUMA_NAME }}
+spec:
+  replicas: {{ KUMA_REPLICAS }}
+  template:
+    metadata:
+      labels:
+        app: {{ KUMA_NAME }}
+    spec:
+      volumes:
+        - name: kuma-fs
+          persistentVolumeClaim:
+            claimName: pv-claim-for-kuma
+      containers:
+        - name: {{ KUMA_NAME }}
+          image: "quay.io/mozmar/kuma:{{ KUMA_GIT_COMMIT_SHORT }}"
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - mountPath: /data/www
+              name: kuma-fs
+          command:
+            - gunicorn
+          args:
+            - "-w=4"
+            - "--bind=0.0.0.0"
+            - "--access-logfile=-"
+            - "--timeout=120"
+            - "kuma.wsgi:application"
+          ports:
+            - containerPort: 8000
+          env:
+            {% include 'kuma-env.yaml.j2' %}
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8000
+            initialDelaySeconds: 10
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /readiness
+              port: 8000
+            initialDelaySeconds: 10
+            periodSeconds: 3
+          resources:
+            requests:
+              cpu: 100m
+              memory: 256Mi
+            limits:
+              cpu: 16
```

Comment:
K8s won't be able to allocate 16 virtual CPU's on the nodes we use. Since our worker nodes are typically [m4.xlarge](https://aws.amazon.com/ec2/instance-types/) for worker nodes, 2-3 should probably be used here. Also, each node needs to run monitoring, kube-system, and other services that will need a slice of the CPU as well.